src/validationplots/tests_to_integrate.jl

```diff
@@ -0,0 +1,95 @@
+using Extremes, DataFrames, Gadfly, Test, Distributions
```

Comment:
Do you want me to integrate the tests before we merge?

Reply:
Not necessarily you :) These tests may be implemented in the near future.